addons/web/static/src/views/calendar/calendar_year/calendar_year_renderer.js

```diff
@@ -183,4 +201,11 @@ export class CalendarYearRenderer extends Component {
     onWindowResize() {
         this.updateSize();
     }
+
+    onEventContent(info) {
+        // Remove the title on the background event like in FCv4
+        if (info.event.display.match(/background/)) {
```

Comment:
Could `display` be a non-string (like null, undefined...) ?
Also, just use `String.includes()` instead of a regular expression that need to be compiled each time ?